src/settings/FinePolicy/FinePolicyForm.test.js

```diff
@@ -68,7 +68,7 @@ describe('FinePolicyForm', () => {
   const policyForTest = new FinePolicy(mockedInitialValues);
   const defaultProps = {
     stripes: mockedStripes,
-    pristine: true,
+    pristine: false,
```

Comment:
The default value of pristine in a `Form` is `true`, which means the form values are not changed. It is not clear why it becomes false.

Reply:
Fixed